ballerina/client.bal

```diff
@@ -13,3 +13,2824 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
+import shopify.oas;
+
+import ballerina/http;
+
+# This is the public Shopify REST API.
+public isolated client class Client {
+    final oas:Client generatedClient;
+    final string accessToken;
+    final string apiVersion;
+
+    # Gets invoked to initialize the `connector`.
+    #
+    # + config - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(string accessToken, string apiVersion, ConnectionConfig config, string serviceUrl) returns error? {
+        self.accessToken = accessToken;
+        self.apiVersion = apiVersion;
```

Comment:
The backend only support specific version or a set of specific versions right? If we stick to the latest one (assuming it will not be changed in the future) then we could remove this wrapper client

Reply:
Fixed in [2a5c295](https://github.com/ballerina-platform/module-ballerinax-shopify/pull/7/commits/2a5c295c5965f77e984fa5f4b0224ac1955f3108)